css/styles.css

```diff
@@ -1,22 +1,25 @@
-main {
-  max-width: 38rem;
-  padding: 1.5rem;
-  margin: auto;
-  text-align: center;
+:root {
+  --red: #e81c4f;
+  --h-kerning: 0.2em;
+  --max-width: 38em;
 }
 
 body {
   margin: 0 auto;
-  max-width: 50em;
+  max-width: var(--max-width);
```

Comment:
TIL you can do this in modern CSS

Reply:
We can also have if-statements, fallback values, and can set them in JS. Also, the values cascade.
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

What a time to be alive.